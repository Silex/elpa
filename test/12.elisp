;;; Relint test file 12          -*- emacs-lisp -*-

;; Test auto-mode-alist

(define-generic-mode my-mode
  nil
  nil
  nil
  '(".aa\\'" "\\.bb$" "^cc.*dd"))

(add-to-list 'auto-mode-alist '("\\.ee$" . some-mode))
(push '(".ff\\'" . some-mode) auto-mode-alist)
(setq auto-mode-alist (cons '(".gg\\'" . some-mode) auto-mode-alist))
(setq auto-mode-alist (append '((".hh\\'" . some-mode)
                                (".ii\\'" . some-mode))
                              auto-mode-alist))
