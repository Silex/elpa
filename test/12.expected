12.elisp:9:6: Possibly unescaped `.' in file-matching regexp (pos 0)
  ".aa\\'"
   ^
12.elisp:9:20: Use \' instead of $ in file-matching regexp (pos 4)
  "\\.bb$"
   .....^
12.elisp:9:24: Use \` instead of ^ in file-matching regexp (pos 0)
  "^cc.*dd"
   ^
12.elisp:11:39: Use \' instead of $ in file-matching regexp (pos 4)
  "\\.ee$"
   .....^
12.elisp:12:10: Possibly unescaped `.' in file-matching regexp (pos 0)
  ".ff\\'"
   ^
12.elisp:13:32: Possibly unescaped `.' in file-matching regexp (pos 0)
  ".gg\\'"
   ^
12.elisp:14:35: Possibly unescaped `.' in file-matching regexp (pos 0)
  ".hh\\'"
   ^
12.elisp:15:35: Possibly unescaped `.' in file-matching regexp (pos 0)
  ".ii\\'"
   ^
